### ISSUE TITLE
Problem about converting imageset format

### ISSUE THREAD

**Issue by charliess123** [conner99/caffe#36]
Hi conner99 @conner99 
I'm trying to convert VOC0712 imageset into lmdb. And Because I haven't install python yet, it occurs some warning about unexisting python instruction when I 'm trying to use "create_data.bat" to convert my imageset format. 
So my question is that necessary to install python to finish my imageset format convertion.
Look forward to your reply.

**Comment by GreenJenEr** [comment]
yes,you must install python or acaconda. As for acaconda ,it is convenient for you to play python!
